src/algebra/group/defs.lean

```diff
@@ -290,16 +290,73 @@ class cancel_comm_monoid (M : Type u) extends left_cancel_comm_monoid M, right_c
 
 end cancel_monoid
 
-/-- A `group` is a `monoid` with an operation `⁻¹` satisfying `a⁻¹ * a = 1`. -/
-@[protect_proj, ancestor monoid has_inv]
-class group (α : Type u) extends monoid α, has_inv α :=
-(mul_left_inv : ∀ a : α, a⁻¹ * a = 1)
-/-- An `add_group` is an `add_monoid` with a unary `-` satisfying `-a + a = 0`. -/
-@[protect_proj, ancestor add_monoid has_neg]
-class add_group (α : Type u) extends add_monoid α, has_neg α :=
-(add_left_neg : ∀ a : α, -a + a = 0)
+/-- `div_sub_tac` proves `∀ a b, a / b = a * b⁻¹` or `∀ a b, a - b = a + -b`
+if it holds by definition. -/
+meta def div_sub_tac : tactic unit := `[intros; refl]
```

Comment:
I think this prevents it occupying a name in the global namespace?
```suggestion
private meta def div_sub_tac : tactic unit := `[intros; refl]
```

Reply:
I'm inclined to keep it non-private since I expect it is more likely to be reused rather than cause name collisions. (The corresponding `order_laws_tac` for `preorder` is also non-private.)

Reply:
frankly these definitions don't need to be namespaced at all, it can just be called `try_refl_tac` instead of having multiple identical tactics. (Also the position after `.` does not do name resolution, so you have to spell out all the namespaces, and if you used `private` it wouldn't be accessible at all.)